Packs/Edgescan/Integrations/Edgescan/README.md

```diff
@@ -0,0 +1,5110 @@
+    | Fetch incidents |  | False |
+    | Incident type |  | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### edgescan-host-get-hosts
+***
+Get a list of all hosts
+
+
+#### Base Command
+
+`edgescan-host-get-hosts`
+#### Input
+
+There are no input arguments for this command.
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Edgescan.HostGetHosts | Unknown | List of all hosts | 
+
+
+#### Command Example
+```!edgescan-host-get-hosts```
+
+#### Human Readable Output
+
+>### Hosts
+>**No entries.**
+
+
+### edgescan-host-get
+***
+Get detailed information about a host.
+
+
+#### Base Command
+
+`edgescan-host-get`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| id | The host id. | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Edgescan.HostGet | Unknown | Detailed host information | 
+
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
+
+
+
+### edgescan-host-get-export
+***
+Get a list of hosts in export format.
+
+
+#### Base Command
+
+`edgescan-host-get-export`
+#### Input
+
+There are no input arguments for this command.
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Edgescan.HostGetExport | Unknown | Export host information | 
+
+
+#### Command Example
+```!edgescan-host-get-export```
+
+#### Human Readable Output
+
+>### Hosts export
+>**No entries.**
+
+
+### edgescan-host-get-query
+***
+Get a list of hosts by query
+
+
+#### Base Command
+
+`edgescan-host-get-query`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| asset_id | The asset ID. | Optional | 
+| os_name | The Operating System name. | Optional | 
+| label | The asset label. | Optional | 
+| status | The asset status. | Optional | 
+| id | The host id. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Edgescan.HostGetQuery | Unknown | The result of a host query | 
+
+
+#### Command Example
+```!edgescan-host-get-query os_name=Linux asset_id=5 id=6 ```
+
+#### Human Readable Output
+
+>### Hosts query
+>**No entries.**
+
+
+### edgescan-host-update
+***
+Update a host
+
+
+#### Base Command
+
+`edgescan-host-update`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| label | The host label. | Optional | 
+| id | The host id to update. | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Edgescan.HostUpdate | Unknown | Information returned after host update | 
+
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
```

Comment:
Missing few more sections to remove. 
Please go over the entire file.
There are some commands with no outputs and no command examples